transforms/code/repo_level_ordering/ray/src/dpk_repo_level_order/internal/repo_level_wrappers.py

```diff
@@ -38,50 +47,72 @@ def get_sorting_func(
         sort_by = semantic_sort_normalised
         logger.info("normalised semantic sort enabled")
     else:
-        sort_by = sort_by_path
-        logger.info(f"sort by path enabled")
+        sort_by = default_sort
+        logger.info("sort by path enabled")
 
-    def sorter(table: pa.Table) -> pa.Table:
+    def sorter(table: pa.Table, file_name: str) -> pa.Table:
+        if table.num_rows < 2:
+            logger.info(f"Not enough rows to sort for {file_name}. Skip sorting.")
+            return table
         df = table.to_pandas()
-        sorted_df = sort_by(df=df, logger=logger, title_column_name=title_column_name)
+        try:
+            sorted_df = sort_by(
+                df=df, logger=logger, title_column_name=title_column_name, language_column_name=language_column_name
+            )
+        except FunctionTimedOut as e:
+            logger.error(
+                f"We got an exception while sorting [{file_name}].\n Exception: {e.__class__.__name__}.\n Falling back to default sort_by_path"
```

Comment:
can we rephrase this error msg like: "Exception while sorting [{file_name}].\n Exception: {e.__class__.__name__}.\n Falling back to default sort_by_path"